match/src/Piipan.Match.State/MatchResponse.cs

```diff
@@ -57,6 +57,10 @@ public PiiRecord()
         [JsonProperty("participant_id")]
         public string ParticipantId { get; set; }
 
+        [JsonProperty("benefits_end_month")]
+        [JsonConverter(typeof(DateMonthConverter))]
+        public DateTime BenefitsEndMonth { get; set; }
```

Comment:
I think your problem might be related to making this nullable. 
```suggestion
        public DateTime? BenefitsEndMonth { get; set; }
```

In general, we probably need to carefully consider null versus empty string and revisit ParticipantId and any other new fields that are optional. Similarly, on the CSV input side, we should be trimming white space and specifying null for empty string in database.

Reply:
Yep, that did it! I added some integration tests to cover that. I can make an issue to evaluate our overall handling of nulls vs empty strings.

Reply:
I made it nullable in both state and orchestrator responses. Manually tested and looks gtg.